packages/relay/src/RelayServer.ts

```diff
@@ -805,22 +801,14 @@ latestBlock timestamp   | ${latestBlock.timestamp}
   }
 
   isReady (): boolean {
-    if (this.lastSuccessfulRounds < this.config.successfulRoundsForReady) {
-      return false
-    }
     return this.ready
   }
 
   setReadyState (isReady: boolean): void {
     if (this.isReady() !== isReady) {
       const now = Date.now()
       if (isReady) {
-        if (this.lastSuccessfulRounds < this.config.successfulRoundsForReady) {
-          const roundsUntilReady = this.config.successfulRoundsForReady - this.lastSuccessfulRounds
-          this.logger.warn(chalk.yellow(`Relayer state: almost READY (in ${roundsUntilReady} rounds)`))
-        } else {
-          this.logger.warn(chalk.greenBright('Relayer state: READY'))
-        }
+        this.logger.warn(chalk.greenBright('Relayer state: READY'))
```

Comment:
Does it solve anything? Like, I get it that the relay turns into 'ready' faster but the idea behind 'successfulRoundsForReady' was that if there is something wrong with an RPC to let it resolve itself.

Reply:
indeed. instead of removing the roundsForRedy, maybe configure it to 0 (or 1) would get the same effect?